__tests__/integration.test.js

```diff
@@ -75,4 +74,65 @@ beforeEach(() => {
           expect(response.body.message).toBe('404 - Bad Request');
         });
     });
-  });
\ No newline at end of file
+  });
+
+  describe("GET /api/articles", () => {
+    test("should return a 200 status and array of article objects", () => {
+      return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+        const articles = response.body.articles;
+        expect(Array.isArray(articles)).toBe(true);
+      });
+    })
+    test.only("should return an array containing expected properties as well as a comment_count", () => {
+      return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+        const articles = response.body.articles;
+        expect(articles.length).toBe(13)
+        articles.forEach((article) => {
+          expect(article).toHaveProperty("author");
+          expect(article).toHaveProperty("title");
+          expect(article).toHaveProperty("article_id");
+          expect(article).toHaveProperty("topic");
+          expect(article).toHaveProperty("created_at");
+          expect(article).toHaveProperty("votes");
+          expect(article).toHaveProperty("article_img_url");
+          expect(article).toHaveProperty("comment_count");
+      })  
+    })
+
+  })
+    test("should return article objects which have the correct data type values", () => {
+      return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+        const articles = response.body.articles;
+        articles.forEach((article) => {
+          expect(article).toMatchObject({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(String)
+        })
```

Comment:
Please merge these tests into one or do a length check for each. 
Every time you do a forEach you need to check the length 👍